Libraries/Components/Keyboard/KeyboardAvoidingView.js

```diff
@@ -184,7 +189,10 @@ const KeyboardAvoidingView = React.createClass({
         );
 
       case 'padding':
-        const paddingStyle = {paddingBottom: this.state.bottom};
+        let paddingStyle = {paddingBottom: this.state.bottom};
+        if (Platform.OS !== 'ios' && Platform.Version < 19) {
```

Comment:
If this is Android specific, then check for Platform.OS being Android. You can't assume that anything that is not iOS will be Android.